list/listService.ts

```diff
@@ -1,12 +1,35 @@
-import uuid from "uuid";
-
+import listModel from './listModel';
+import Contact from '../contact/сontactService';
 class List {
   private uuid: string;
   private name: string;
 
-  public constructor(name: string) {
-    this.uuid = uuid.v4();
+  public constructor(name: string = '', uuid: string = '') {
     this.name = name;
+    this.uuid = uuid;
+  }
+
+  static async getContactsOfListByListId(listId: string): Promise<string> {
+    return await Contact.getContactsOfList(listId);
+  }
+
+  public async addNewList(): Promise<void> {
+    const name = this.name;
+    await listModel.create({ name });
+  }
+
+  public async addContactToList(): Promise<void> {
```

Comment:
You should provide contact id in this function and add it to the list

Reply:
The values were provided as parameters to the function